### PR TITLE
Re-cookiecut, move tests to root level

### PR DESCRIPTION
**Pull request by bobleesj**
Closes https://github.com/diffpy/diffpy.pdfgui/issues/209, https://github.com/diffpy/diffpy.pdfgui/issues/207, https://github.com/diffpy/diffpy.pdfgui/issues/175, https://github.com/diffpy/diffpy.pdfgui/issues/162, https://github.com/diffpy/diffpy.pdfgui/issues/37

### CODE REVIEW

File: doc/make.bat
Comment (bobleesj) [inline doc/make.bat:1]: Docs tested locally.

**Review by bobleesj — COMMENTED** [review]
Ready for review

**Comment by bobleesj** [comment]
Seems like there is a typo of `xport` instead of `export` in https://github.com/Billingegroup/release-scripts/blob/main/.github/workflows/_tests-on-pr.yml

```
      - name: Validate ${{ inputs.project }}
        run: |
          if ${{ inputs.headless }}; then
            xport DISPLAY=:99
```

File: requirements/run.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-numpy < 2.0.0  # Need to fix deprecations before 2.0.0 compat
-wxpython
-diffpy.pdffit2
-diffpy.structure
-diffpy.utils
-matplotlib-base
+numpy
```
Comment (bobleesj) [inline requirements/run.txt:1]: Removed pinning since the latest release on conda-forge (diffpy.structure, diffpy.pdffit2 have no numpy pinning)

Comment (bobleesj) [reply requirements/run.txt:1]: In `proproject.toml`, however, still maintained as discussed with the wx problem. 

```
        'Programming Language :: Python :: 3.10',
        'Programming Language :: Python :: 3.11',
```

**Comment by bobleesj** [comment]
@sbillinge Just noticed that `xport` to `export` typo must be also fixed in `v0` of the release script as well as `main`. Created a PR: https://github.com/Billingegroup/release-scripts/pull/46

**Comment by sbillinge** [comment]
I reran the tests after merging main into v0  and the tests failed in the same way.  @bobleesj Pls could you take a quick look.  I think it is probably something very simple.

**Comment by bobleesj** [comment]
(Still running tests using my own local fork) I will ping @sbillinge once ready for review.

**Comment by bobleesj** [comment]
I think it's ready @sbillinge  Had to pin numpy < 2 but made an issue 

https://github.com/diffpy/diffpy.pdfgui/issues/212

**Pull request merged**